internal/providers/terraform/azure/testdata/vpn_gateway_connection_test/vpn_gateway_connection_test.usage.yml

```diff
@@ -0,0 +1,13 @@
+version: 0.1
```

Comment:
There shouldn't be a usage component to this resource.  Each _connection counts as one tunnel.